docs/v2/guides/internationalization.md

```diff
@@ -41,26 +45,41 @@ You can use sprintf-type placeholders, using the `format` filter:
 **WordPress:**
 
 ```html
-<p class="entry-meta"><?php printf( __('Posted on %s', 'my-text-domain'), $posted_on_date ) ?></p>
+<p class="entry-meta">
+    <?php printf( __('Posted on %s', 'my-text-domain'), $posted_on_date ) ?>
+</p>
 ```
 
 **Timber:**
 
 ```twig
-<p class="entry-meta">{{ __('Posted on %s', 'my-text-domain')|format(posted_on_date) }}</p>
+<p class="entry-meta">
+    {# Translators: The placeholder will be replaced with the localized post date #}
+    {{ __('Posted on %s', 'my-text-domain')|format(posted_on_date) }}
+</p>
 ```
 
 If you want to use the `sprintf` function in Twig, you have to [add it yourself](https://timber.github.io/docs/v2/guides/functions/#make-functions-available-in-twig).
 
 ## Generating localization files
 
-To generate `.pot`, `.po` and `.mo` files, you need a tool that supports parsing Twig files to detect all your translations. While there are a lot of tools that can parse PHP files, the solution that works best for Twig files is [Poedit](https://poedit.net/).
+### wp-i18n-twig (recommended)
 
-### Generating l10n files with Poedit 2
+The Timber team provides a [dedicated WP-CLI package](https://github.com/timber/wp-i18n-twig) that takes care of extracting translations from Twig files.
+
+This main benefit over other tools is that it perfectly integrates with the existing `wp i18n` command, making `.pot` file generation simple and easy.
+
+Note that the package isn't Timber specific, it will work with any Twig implentation making use of WordPress translation functions.
```

Comment:
```suggestion
Note that the package isn't Timber specific, it will work with any Twig implementation that also provides WordPress translation functions.
```